src/Features/CSharp/Portable/InitializeParameter/CSharpInitializeMemberFromParameterCodeRefactoringProvider.cs

```diff
@@ -35,5 +35,8 @@ protected override void InsertStatement(SyntaxEditor editor, SyntaxNode function
 
         protected override bool IsImplicitConversion(Compilation compilation, ITypeSymbol source, ITypeSymbol destination)
             => InitializeParameterHelpers.IsImplicitConversion(compilation, source, destination);
+
+        // We always generate private fields. Since "private" is the default accessibility for fields in C#, we do not need an accessibility modifier.
+        protected override Accessibility CreateFieldHelper(Document document, SyntaxNode functionDeclaration, SemanticModel model, CancellationToken cancellationToken) => Accessibility.NotApplicable;
```

Comment:
nit: wrap => to the next line, like the method above.

Reply:
Someting aobut this name doesn't make sense to me.  I think it woul dbe better as DetermineDesiredFieldAccessibility.